src/storage/v2/vertex_accessor.cpp

```diff
@@ -26,6 +28,7 @@
 #include "storage/v2/storage.hpp"
 #include "storage/v2/vertex_info_cache.hpp"
 #include "storage/v2/vertex_info_helpers.hpp"
+#include "text_search.hpp"
```

Comment:
if you still need this conclusion (I think you do), import as storage/.../text_search.hpp

Reply:
Fixed (this #include wasn’t needed)